app/views/asset_notification/upload_notification.text.erb

```diff
@@ -1,26 +1,26 @@
 Hi!
 
 <% if @description.present? %>
-<%= @name %> just uploaded a new track on alonetone. <%= @exclamation %>! Here is what they had to say about it:
+New track by <%= @name.capitalize %> was just uploaded to alonetone! <%= @exclamation %>! Here is what they had to say about it:
 
 <%= @description %>
 
 - - - - - - - - - - - - - - - -
 <% else %>
-<%= @name %> just uploaded a new track on alonetone. <%= @exclamation %>!
+New track by <%= @name.capitalize %> was just uploaded to alonetone! <%= @exclamation %>!
 
-It's so piping hot we don't even know the juicy details yet. <%= @name %> is still typing them in.
+It's so piping hot we don't even know the juicy details yet. <%= @name.capitalize %> is still typing them in.
 <% end %>
 
-Listen to it at alonetone:
+To listen to <%= @title %> on alonetone:
```

Comment:
Didn't want to break the `flow` of the email by sticking the title somewhere up top. Would this place be enough you think? Or we can do something more explicit in the first/second sentence.